index.html

```diff
@@ -0,0 +1,480 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Technical Document</title>
+    <link rel="stylesheet" href="./styles.css"
+  </head>
+  <body>
+    <nav id="navbar">
+      <header>JS Documentation</header>
```

Comment:
For good practice and proper code organization, the nav element should be inside the header element